gnowsys-ndf/gnowsys_ndf/ndf/models.py

```diff
@@ -293,77 +293,95 @@ class Node(DjangoDocument):
     use_dot_notation = True
 
 
-    def fill_node_values(self, request=HttpRequest(), **kwargs):
+    def fill_node_values(self, request, **kwargs):
 
```

Comment:
Provide some default argument for **`request`** parameter in `fill_node_values` method signature.